### PR TITLE
📖 Update Cherry-pick template with QA section

### PR DESCRIPTION
**Pull request by rsimha**
This PR adds a new section to the CP template to enable AMP's QA expert to verify changes after they are cherry-picked.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/cherry_pick_template.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ CONDITION: Cherry-picking into LTS. Otherwise, delete.
 
 <_YOUR_REASONS_>
 
+## List the steps to manually verify the changes in this cherry-pick
```
Comment (kristoferbaxter) [inline .github/ISSUE_TEMPLATE/cherry_pick_template.md:52]: Should we include a link to an example of the steps?

**Review by kristoferbaxter — APPROVED** [review]
Question included, but not a blocker.

File: .github/ISSUE_TEMPLATE/cherry_pick_template.md (hunk 1 above)
Comment (rsimha) [reply .github/ISSUE_TEMPLATE/cherry_pick_template.md:52]: Do you have a good example to suggest? I've currently stuck to the convention in the template by including a `<TEST_STEPS>` placeholder for the CP author to fill out.

Comment (rsimha) [reply .github/ISSUE_TEMPLATE/cherry_pick_template.md:52]: Merging for now, will incrementally improve this as developers offer feedback.

**Pull request merged**